spreadsheet.py

```diff
@@ -183,9 +179,10 @@ def append_rows(self, rows, target, first_table_cell="A1"):
         append to happen in the wrong place.
         """
         cell_range = target + "!" + first_table_cell
-        body = { "values": rows }
-        req = self.spreadsheets().values().append(spreadsheetId=self._id, 
-                range=cell_range, valueInputOption='RAW', 
-                insertDataOption='INSERT_ROWS', body=body)        
+        body = {"values": rows}
+        req = self.spreadsheets().values().append(spreadsheetId=self._id,
+                                                  range=cell_range,
+                                                  valueInputOption='RAW',
+                                                  insertDataOption='INSERT_ROWS', # noqa
```

Comment:
For line length, we want to actually fix the problem not ignore it.  So we can use the other method of indentation, like this:

```
req = self.spreadsheets().values().append(
    spreadsheetId=self._id,
    range=cell_range, 
    valueInputOption='RAW',
    insertDataOption='INSERT_ROWS', 
    body=body)
```
So you go to a new line after the `(`, and the put 4 spaces in front of each line.